.neutrinorc.js

```diff
@@ -52,20 +142,19 @@ module.exports = {
           },
         },
       },
-      eslint: {
-        rules: {
-          // This has been set in the latest Airbnb preset, but has not been
-          // released yet.
-          'react/no-did-mount-set-state': 'off',
-        }
-      }
+      env: [
```

Comment:
If it helps, this `env` option now also supports the object-form that can take defaults for each variable.